src/styles/blocks/card.scss

```diff
@@ -0,0 +1,92 @@
+.card {
+  width: 200px;
+  padding: 29px 14px 14px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: $standard-color;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin-bottom: 45px;
+  }
+
+  &__name {
+    margin: 0 0 6px;
+    font-weight: $fw-semibold;
+    font-size: $fs-md;
+    line-height: 15px;
+  }
+
+  &__code {
+    margin: 0 0 15px;
```

Comment:
there is information in checklist, that we should use margin/padding-[direction], if we have only one